### ISSUE TITLE
what's the point of this exporter?

### ISSUE THREAD

**Issue by redthnb** [hiboxsystems/redis_sentinel_exporter#4]
Can you explain please what is the point of this exporter? I think that I don't see the full picture.

https://github.com/oliver006/redis_exporter has sentinel metrics already.

```
		"sentinel_master_ok_sentinels":
		"sentinel_master_ok_slaves":
		"sentinel_master_sentinels":
		"sentinel_master_slaves":
		"sentinel_master_status":
		"sentinel_masters":
		"sentinel_running_scripts":
		"sentinel_scripts_queue_length":
		"sentinel_simulate_failure_flags":
		"sentinel_tilt":
```

**Comment by perlun** [comment]
Sorry for the (very!) late reply on this one @redthnb, @yurymuski and @Gaozizhong. AFAIK, this exporter added some value, but I have switched jobs since I was involved with creating it so I'm not sure of the exact details here.

@slovdahl, is this exporter still used? If not, we should perhaps mark the repo as archived and add a note to the README. (We should be careful to not remove it, since it's still Free Software and can be useful)

**Comment by slovdahl** [comment]
It's still in use. Yeah, it's possible that the same metrics are provided by other exporters as well today, but IIRC that was not the case at the time when we started using it.

And FWIW, we are not the original creators of this exporter. The code came from https://github.com/leominov/redis_sentinel_exporter that no longer exists.

**Comment by perlun** [comment]
> And FWIW, we are not the original creators of this exporter. The code came from https://github.com/leominov/redis_sentinel_exporter that no longer exists.

Yeah, that's a good point. 👍 I wrote some details about that here in this issue in fact:
https://github.com/hiboxsystems/redis_sentinel_exporter/issues/1#issuecomment-1055246366
